src/helpers/transformTokenValue.ts

```diff
@@ -3,6 +3,8 @@ import {Transaction} from 'src/interfaces/transaction';
 enum TokenID {
```

Comment:
this helper will replaced on [PR-313](https://github.com/myriadsocial/myriad-web/pull/313) right?

Reply:
yes

Reply:
ok thank u